Cognite.Simulator.Utils/FileLibrary.cs

```diff
@@ -257,7 +265,9 @@ await FindFiles(true, token)
                         if (response.Any() && response.First().DownloadUrl != null)
                         {
                             var uri = response.First().DownloadUrl;
-                            var filename = Path.Combine(_modelFolder, $"{file.CdfId}.{file.GetExtension()}");
+                            var storageFolder = Path.Combine(_modelFolder, $"{file.CdfId}");
```

Comment:
I'm worried about backwards compatibility with previous connector versions. For instance, when updating the connector, it will not be able to delete the files saved by the previous version.

Reply:
So, in my local testing, if you just delete the `state.db` and `files` folder when you install this update, everything remains fine. If you dont delete the `files` folder but do delete `state.db`, all the previously saved files still remain in the folder, but they wont be used anymore. I have not tested without deletion of `state.db`, but this will be the worst case scenario.

Reply:
I suggest testing upgrading the connector as the user would. We cannot guarantee that they will delete the state and the files. Typical case is just installing a new version while the previous one is already installed, with files downloaded and state already there.